tavla/app/(admin)/oversikt/utils/index.ts

```diff
@@ -7,3 +7,8 @@ export const dataColors = {
     lilac: 'var(--data-visualization-lilac)',
     spring: 'var(--data-visualization-spring)',
 }
+
+export interface QueryParam {
+    key: string
```

Comment:
Kunne man laget noen enums/konstanter ut av de ulike `key` man kan sette? Eks: 

```
QueryKeys = {
Opprett = 'opprett'
Flytt = 'flytt'
Slett = 'Slett'
Id = 'id"
}
```
Jeg kan se for meg at det da er lettere videre å ikke introdusere 100 nye typer, men at man klarer å si hva som finnes der fra før av? 🤔 Ref at det er en del utskiftninger

Da blir `key` av type `QueryKeys` f. eks. Usikker om det er best practise.